testcontainers/src/runners/sync_runner.rs

```diff
@@ -0,0 +1,234 @@
+use crate::{Container, Image, RunnableImage};
+
+/// Helper trait to start containers synchronously.
+pub trait SyncRunner<I: Image> {
+    /// Starts the container and returns an instance of `Container`.
+    fn start(self) -> Container<I>;
+}
+
+impl<T, I> SyncRunner<I> for T
+where
+    T: Into<RunnableImage<I>> + Send,
+    I: Image,
+{
+    fn start(self) -> Container<I> {
+        let rt = tokio::runtime::Builder::new_current_thread()
```

Comment:
I personally like to avoid using abbreviations (except for widely known ones like e.g. `HTTP`) for more clarity inside the code.

Reply:
I don't usually mind commonly used acronyms, but let me change this for clarity